tests/suites/test_suite_x509write.data

```diff
@@ -265,3 +265,9 @@ mbedtls_x509_string_to_names:"C=NL, 2.5.4.10.234.532=#0C084F6666737061726B, OU=P
 
 Check max serial length
 x509_set_serial_check:
+
+Check max extension length (Max-1)
+x509_set_extension_length_check:0xFFFFFFFE
+
+Check max extension length (Max)
+x509_set_extension_length_check:0xFFFFFFFF
```

Comment:
Nit: I think our CI will object to the lack of newline at end of file.

Reply:
I updated it.